buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -66,7 +66,7 @@
         "definitionType": "task"
       },
       "inputs": {
-        "SourceFolder": "pkg",
+        "SourceFolder": "$(PB_RepoName)/pkg",
```

Comment:
I haven't tested this. I'm pretty confident it will work because it matches what we use for build.sh: https://github.com/dotnet/core-setup/blob/e01befc220e46313417e8ff226ab371930f94370/buildpipeline/Core-Setup-CrossBuild.json#L127

    "scriptPath": "$(PB_RepoName)/scripts/docker-as-current-user.sh",